modules/eventing/eventing.go

```diff
@@ -29,6 +30,7 @@ type Module struct {
 	functions *functions.Module
 	adminMan  *admin.Manager
 	syncMan   *syncman.Manager
+	fileStore *filestore.Module
```

Comment:
Do we really need this here?

Reply:
yes, how can i call doesExist function defined in filestore module?

Reply:
Got it